Extensions/Video/videoruntimeobject-pixi-renderer.js

```diff
@@ -0,0 +1,238 @@
+/**
+ * The PIXI.js renderer for the VideoRuntimeObject.
+ *
+ * @class VideoRuntimeObjectPixiRenderer
+ * @constructor
+ * @param {gdjs.VideoRuntimeObject} runtimeObject The object to render
+ * @param {gdjs.RuntimeScene} runtimeScene The gdjs.RuntimeScene in which the object is
+ */
+gdjs.VideoRuntimeObjectPixiRenderer = function(runtimeObject, runtimeScene) {
+  this._object = runtimeObject; // Keep a reference to the object to read from it later.
+
+  // Here we're going to create a video text as an example.
+  if (this._pixiObject === undefined) {
+    this._textureVideo = new PIXI.Texture.fromVideo(
+      "C:/Users/RTX-Bouh/Desktop/test_video_GD.mp4"
+    );
+
+    //Setup the PIXI object:
+    this._pixiObject = new PIXI.Sprite(this._textureVideo);
+
+    this._pixiObject._texture.baseTexture.source.pause();
+  }
```

Comment:
If `this._pixiObject` is already defined, this means that the renderer is being "recycled" (when objects are destroyed, they are removed from the scene, put in a cache but not deleted, and then reused to avoid creating/deleting a lot of objects). So the renderer constructor might be called with an existing video - in which case you must reset it to initial value.

```
else {
  this._pixiObject._texture.baseTexture.source.currentTime = 0;
  this._pixiObject._texture.baseTexture.source.pause();
}
```